pkg/resource/data_quality_job_definition/testdata/data_quality_job_definition/v1alpha1/dqjd_invalid_create_attempted.yaml

```diff
@@ -44,6 +44,7 @@ status:
   ackResourceMetadata:
     ownerAccountID: ""
   conditions:
-  - message: The job definition name must start with an alphanumeric character.
+  - message: "InvalidParameterValue: The job definition name must start with an alphanumeric
+      character.\n\tstatus code: 0, request id: "
```

Comment:
These error messages say name must start with alphanumeric character but the provided name does start with a alphanumeric character that character being “i” am I misunderstanding these tests ?

Reply:
aah, the error message is dummy but let me change it for future purposes

Reply:
done